circle.yml

```diff
@@ -23,6 +23,12 @@ deployment:
   production:
     branch: release
     commands:
+      # Build and push Docker image tagged latest and tagged with commit descriptor
+      - docker build -t interledgerjs/ilp-kit:latest .
+      - docker tag interledgerjs/ilp-kit:latest interledgerjs/ilp-kit:"$(git describe --tags)"
+      - sed "s/<AUTH>/${DOCKER_TOKEN}/" < "dockercfg-template" > ~/.dockercfg
```

Comment:
If you just have the string `${DOCKER_TOKEN}` in the dockercfg-template, I think it'll grab the env var automatically

Reply:
We can address that in a follow-up issue, this is just to restore what was there before.